pom.xml

```diff
@@ -102,7 +99,6 @@
     <dependency>
       <groupId>org.jenkinsci.plugins</groupId>
       <artifactId>kubernetes-credentials</artifactId>
-      <version>${jenkins-kubernetes-credentials.version}</version>
```

Comment:
Why did you decided to _unpin_ all those dependencies ? Doesn't it mean that Jenkins will be less restrictive when deciding which plugins it can update / start with, and this would lead to runtime errors instead of update/boot errors ?

Reply:
using managed version from group io.jenkins.tools.bom - bom-2.401.x